app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
+class SavedClaim::VeteranReadinessEmploymentClaim < SavedClaim
+  include SentryLogging
+  FORM = '28-1900'
+  class VREError < StandardError; end
+
+  def add_claimant_info(user)
+    return if form.blank?
+
+    updated_form = parsed_form
+
+    updated_form['veteranInformation'] = {
+      'fullName' => {
+        'first' => user.first_name,
+        'middle' => user.middle_name || '',
+        'last' => user.last_name
+      },
+      'ssn' => user.ssn,
+      'VAFileNumber' => veteran_va_file_number(user),
+      'pid' => user.participant_id,
+      'edipi' => user.edipi,
+      'vet360ID' => user.vet360_id,
+      'dob' => user.birth_date
+    }
+
+    update(form: updated_form.to_json)
+  end
+
+  def send_to_vre
+    conn = Faraday.new(url: Settings.vre.base_url)
+
+    response = conn.post do |req|
+      req.url Settings.vre.ch_31_endpoint
+      req.headers['Authorization'] = "Bearer #{get_token}"
+      req.headers['Content-Type'] = 'application/json'
+      req.body = format_payload_for_vre
+    end
+
+    response_body = JSON.parse(response.body)
+    return true if response_body['ErrorOccurred'] == false
+
+    raise VREError
+  rescue VREError => e
+    log_exception_to_sentry(
+      e,
+      {
+        error_message: response_body['ErrorMessage']
+      },
+      { team: 'vfs-ebenefits' }
+    )
+  end
```

Comment:
This shouldn't be in the model - it looks like what you're trying to do is instantiate a connection to the VRE endpoint - in which case I'd like to see a service class created in `lib/**/*`, check out that folder for lots of examples on the pattern. 

Stuffing connection logic into a model isn't very Rails-y and confuses the purpose of the class in my experience, which in this case should represent the data for a saved claim, not act as an interface to the external service.

Reply:
This totally makes sense. I had analysis paralysis for where this should go. Figured I would get the code working, throw it in here and see what the reviewer(you) thought.

This has been changed.